### PR TITLE
[kddockwidgets] new port

### PR DESCRIPTION
**Pull request by Neumann-A**

### CODE REVIEW

**Comment by dg0yt** [comment]
Note that there is #19548.

**Comment by Neumann-A** [comment]
> Note that there is #19548.

yeah but that one is probably never going to finish and is the qt5 version.

File: ports/kddockwidgets/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    list(APPEND _qarg_OPTIONS -DQT_HOST_PATH=${CURRENT_HOST_INSTALLED_DIR})
+    list(APPEND _qarg_OPTIONS -DQT_HOST_PATH_CMAKE_DIR:PATH=${CURRENT_HOST_INSTALLED_DIR}/share)
+    if(VCPKG_TARGET_ARCHITECTURE STREQUAL arm64 AND VCPKG_TARGET_IS_WINDOWS) # Remove if PR #16111 is merged
+        list(APPEND _qarg_OPTIONS -DCMAKE_CROSSCOMPILING=ON -DCMAKE_SYSTEM_PROCESSOR:STRING=ARM64 -DCMAKE_SYSTEM_NAME:STRING=Windows)
+    endif()
+endif()
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    PREFER_NINJA
```
Comment (JonLiu1993) [inline ports/kddockwidgets/portfile.cmake:23]:
```suggestion
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the new port!

**Pull request merged**